slides/src/content/component.md

```diff
@@ -226,13 +239,13 @@ export class Child {
 
 ## Quiz (continued)
 
-1. Parent:  
```

Comment:
My editor has a setting to auto-strip EOL whitespace, I'll look at these issues across my PRs thanks